README.md

```diff
@@ -99,10 +99,18 @@ The config file (default file name: `.lintr`) is in [Debian Control Field Format
 - `exclude_start` - a regex pattern to start exclusion range. Default is "# nolint start"
 - `exclude_end` - a regex pattern to end exclusion range. Default is "# nolint end"
 
-An example file that uses 120 character line lengths, excludes a couple of
-files and sets different default exclude regexs follows.
+
+### .lintr File Example
+
+Below is an exmaple .lintr file that uses:
+
+- 120 character line lengths
+- Excludes a couple of files
+- Disables a specific lintr, and; 
+- Sets different default exclude regexes
+
 ```
-linters: with_defaults(line_length_linter(120))
+linters: with_defaults(line_length_linter(120), commented_code_linter=NULL)
```

Comment:
Maybe we could show off how to properly use line breaks since that is very common for the `linters` key?
Also, please adhere to the default R style in this example, i.e. put spaces around `=`.

Reply:
re linebreaks on linters key. I'm guessing here. You'll have to point me to an example.